app/languages/de/user/editUserProfilelabels.json

```diff
@@ -11,12 +11,12 @@
   "manage_offline": "Offlinekarten verwalten",
   "manage_species": "Arten verwalten",
   "activity_logging": "Aktivität aufzeichnen",
-  "activity_logs": "Aktivitätsaufzeichungen",
+  "activity_logs": "Aktivitäten",
```

Comment:
This translation is too long now with the sharing icon behind and got broken into two lines, so I choose a shorter translation. Other language or smaller display might still have this problem.